test/unit/specs/validators/sameAs.spec.js

```diff
@@ -9,6 +9,9 @@ describe('sameAs validator', () => {
   it('should not validate different string', () => {
     expect(sameAs('first')('world', parentVm)).to.be.false
     expect(sameAs('second')('hello', parentVm)).to.be.false
+    expect(sameAs('first')(undefined, parentVm)).to.be.false
+    expect(sameAs('first')(null, parentVm)).to.be.false
```

Comment:
I just want to be sure that the sameAs validator code handles undefined and null values without any runtime errors.